src/js/components/Box/stories/Round.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Grommet, Box, Grid } from 'grommet';
+import { grommet } from '../../../themes';
+
+const RoundBox = () => (
+  <Grommet theme={grommet}>
+    <Box pad="small" gap="small">
+      <Box pad="small" background="brand" round alignSelf="start">
+        true
+      </Box>
+      <Grid columns="small" gap="small">
+        {['xsmall', 'small', 'medium', 'large', 'xlarge', 'full'].map(size => (
+          <Box key={size} pad="large" background="brand" round={{ size }}>
+            {size}
+          </Box>
+        ))}
+      </Grid>
+      <Grid columns="small" gap="small">
```

Comment:
Added this Grid to visually see the radius impact when responsive is false. easy to notice and compare with the dark-4 color.